offchain-modules/packages/x/src/ckb/tx-helper/multisig/deploy.ts

```diff
@@ -151,14 +150,24 @@ function asyncSleep(ms = 0) {
 const main = async () => {
   console.log('\n\n\n---------start init multisig address -----------\n');
   await waitUntilSync();
-  const configPath = './config.json';
-  nconf.env().file({ file: configPath });
-  const config: Config = nconf.get('forceBridge');
-  console.log('config: ', config);
-  await new ForceBridgeCore().init(config);
-  await deploy();
+  const ckbPrivateKey = getFromEnv('CKB_PRIV_KEY');
```

Comment:
maybe we should add an `.env` file example to describe which config should pass in by env var?

Reply:
This script will not be used by other users. And there is an example in `integration-test.sh`. Even someone use it and miss the env var, the error message of this script will be clear enough for him to add it.